### PR TITLE
 upgrade to web pack 3

### PR DESCRIPTION
**Pull request by pi0**
Release notes: https://github.com/webpack/webpack/releases

(Some improvements inspired by https://jsmantra.com/faster-leaner-and-better-next-js-ea40afb94907)

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -90,7 +90,7 @@ const sysSep = _.escapeRegExp(sep)
 const normalize = string => string.replace(reqSep, sysSep)
 
 export function r () {
-  let args = Array.from(arguments)
+  let args = Array.prototype.slice.call(arguments)
```
Comment (Atinux) [inline lib/utils.js]: Why changing this syntax? Is there a good reason to not use `Array.from`?

Comment (pi0) [inline lib/utils.js]: Actually with webPack 3 upgrade new nuxt.js bundle throws errors like this:
```
TypeError: Cannot assign to read only property 'length' of function 'function from() { [native code] }
```

Comment (Atinux) [inline lib/utils.js]: 👍

**Comment by pi0** [comment]
Closing in flavor of tapable branch.

**Pull request closed without merging**